project1/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
```

Comment:
Two things here about the font stylesheet:
1. You can combine them into one request, which is better for your page performance, like so:
```suggestion
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:600|PT+Serif&display=swap" rel="stylesheet">
```
2. The font stylesheet should be loaded _after_ your custom styles, so this should get moved to just _after_ line 12.